modin/config/pubsub.py

```diff
@@ -28,7 +28,7 @@
 )
 
 if TYPE_CHECKING:
-    from .envvars import EnvironmentVariable
+    from modin.config.envvars import EnvironmentVariable
```

Comment:
```suggestion
    from modin.config import EnvironmentVariable
```
?

Reply:
Such imports will lead to cyclical imports.